packages/remix-eslint-config/rules/typescript.js

```diff
@@ -1,54 +1,49 @@
-const OFF = 0;
-const WARN = 1;
-const ERROR = 2;
-
 module.exports = {
-  "no-dupe-class-members": OFF,
-  "no-undef": OFF,
-
-  // Add TypeScript specific rules (and turn off ESLint equivalents)
-  "@typescript-eslint/consistent-type-assertions": WARN,
-  "@typescript-eslint/consistent-type-imports": WARN,
-
-  "no-array-constructor": OFF,
-  "@typescript-eslint/no-array-constructor": WARN,
+  // TODO: These rules might be nice to enable... we should investigate eventually!
+  "@typescript-eslint/ban-ts-comment": "off",
+  "@typescript-eslint/ban-types": "off",
+  "@typescript-eslint/no-empty-function": "off",
+  "@typescript-eslint/no-empty-interface": "off",
+  "@typescript-eslint/no-explicit-any": "off",
+  "@typescript-eslint/no-inferrable-types": "off",
+  "@typescript-eslint/no-namespace": "off",
+  "@typescript-eslint/no-non-null-assertion": "off",
+  "@typescript-eslint/no-var-requires": "off",
+  "no-var": "off",
+  "prefer-rest-params": "off",
 
-  // There is a bug w/ @typescript-eslint/no-duplicate-imports triggered
-  // by multiple imports inside of module declarations. We should reenable
-  // this rule when the bug is fixed.
-  // https://github.com/typescript-eslint/typescript-eslint/issues/3071
-  "no-duplicate-imports": OFF,
-  // "@typescript-eslint/no-duplicate-imports": WARN,
-
-  "no-redeclare": OFF,
-  "@typescript-eslint/no-redeclare": ERROR,
```

Comment:
I don't see this on in the recommended rules either?

Reply:
The rule is pretty much redundant when https://eslint.org/docs/latest/rules/no-var is enabled. I've seen exceedingly few projects use `var` over the last few years, and within them anything that would trigger the rule. Is there a reason you'd want to keep it enabled?

Reply:
Nah - didn't look too deep into these specific rules before, just was looking at things that were net-removed.  Makes sense that it's not useful with `no-var`